app/src/main/java/io/pslab/activity/MultimeterActivity.java

```diff
@@ -406,52 +406,50 @@ public boolean onOptionsItemSelected(MenuItem item) {
                     if (recordData) {
                         item.setIcon(R.drawable.ic_record_white);
                         recordData = false;
-                        CustomSnackBar.showSnackBar(coordinatorLayout, getString(R.string.data_recording_paused), null, null, Snackbar.LENGTH_LONG);
+                        if (isDataRecorded) {
+                            MenuItem item1 = menu.findItem(R.id.record_pause_data);
```

Comment:
There is a different icon for stop. Have a look at other instruments for the icon